ReportingServicesTools/Functions/CatalogItems/Get-RsSubscription.ps1

```diff
@@ -0,0 +1,141 @@
+﻿# Copyright (c) 2016 Microsoft Corporation. All Rights Reserved.
+# Licensed under the MIT License (MIT)
+
+function Get-RsSubscription
+{
+    <#
+        .SYNOPSIS
+            This script retrieves information about subscriptions for a report.
+        
+        .DESCRIPTION
+            This script retrieves information about subscriptions for a report.
+        
+        .PARAMETER Path
+            Specify the path to the report.
+        
+        .PARAMETER ReportServerUri
+            Specify the Report Server URL to your SQL Server Reporting Services Instance.
+            Use the "Connect-RsReportServer" function to set/update a default value.
+        
+        .PARAMETER Credential
+            Specify the password to use when connecting to your SQL Server Reporting Services Instance.
+            Use the "Connect-RsReportServer" function to set/update a default value.
+        
+        .PARAMETER Proxy
+            Report server proxy to use.
+            Use "New-RsWebServiceProxy" to generate a proxy object for reuse.
+            Useful when repeatedly having to connect to multiple different Report Server.
+        
+        .EXAMPLE
+            Get-RsSubscription -Path '/path/to/my/report'
+            Description
+            -----------
+            This command will establish a connection to the Report Server located at http://localhost/reportserver using current user's credentials and retrieve details of subscriptions found at '/path/to/my/report'.
+        
+        .EXAMPLE
+            Get-RsSubscription -ReportServerUri 'http://remote-machine:8080/reportserver_sql16' -Path '/path/to/my/report'
+            Description
+            -----------
+            This command will establish a connection to the Report Server located at http://remote-machine:8080/reportserver_sql16 using current user's credentials and retrieve details of subscriptions found at '/path/to/my/report'.
+    #>
+
+    [cmdletbinding()]
+    param
+    (
+        [Alias('ItemPath')]
```

Comment:
The alias are intended for backward compatibility, for new functions are not required, please remove it